packages/app-extensions/src/actions/modules/sagas.js

```diff
@@ -19,12 +19,14 @@ export function* invokeAction(config, {payload}) {
     const actionHandler = actionHandlers[definition.actionType]
     const response = yield call(actionHandler, definition, selection, parent, params, config)
 
-    if (response.remoteEvents) {
-      yield all(response.remoteEvents.map(remoteEvent => put(remoteEvents.remoteEvent(remoteEvent))))
-    }
+    if (response) {
```

Comment:
i did something simular with the now merged delete action. you can apply this change.